lib/src/screens/metamap/address_screen.dart

```diff
@@ -249,7 +249,7 @@ class _AddressScreenState extends ConsumerState<AddressScreen> {
                     newUser.address = jsonAddress;
                     ref.read(userProvider.notifier).setUser(newUser);
                     try {
-                      await metaMapController.updateUser(user: newUser);
+                      await metaMapController.updateUser(address: jsonAddress, profession: selectedProfession);
```

Comment:
Para que sea mantenible en el futuro, es mejor pasar un Imput Object a la función

```suggestion
                      await metaMapController.updateUser(updateUserInput);
```